### PR TITLE
Keep Attribute Editor multiple values expanded

### PR DESCRIPTION
**Pull request by Auriga2**
### Description of the Change
When the graph selection was changing, or graph was zooming in or out , any expanded node or transaction attributes were contracting to display "Multiple Values", rather than remaining expanded and populating newly selected values. 

### Alternate Designs
A possible extension is to keep the selection of attributes across graphs, and/or retain the selection upon returning to the graph. Currently it works only within a graph and loses the selection if the user leaves and returns a graph.

### Why Should This Be In Core?
Better UX and saves time

### Benefits
Better UX and saves time

### Possible Drawbacks
N/A

### Verification Process

1. Open a graph and the Attribute Editor
2. Select a set of nodes or transactions such that an attribute has more than one value for the selection
3. Expand an attribute with multiple values
4. Change the graph selection such that the chosen attribute still has multiple values and verify the expanded attribute/s do not collapse. 
5. Select a single Node or Transaction and verify the attribute/s collapse. Select multiple Nodes or Transactions and verify the previously viewed attribute/s expand again.
6. Zoom in and out the graph and verify the attributes are still expanded. 

 
### Applicable Issues

https://github.com/constellation-app/constellation/issues/1128

### CODE REVIEW

File: CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeData.java [hunk 1]
```diff
@@ -128,4 +129,18 @@ public boolean isKey() {
...
+     * Should multiple values of this attribute be always expanded .
+     *
+     * @return True if the attribute values should always be expanded, false
+     * otherwise.
+     */
+    public boolean isKeepExpanded() {
+        return isKeepExpanded;
+    }
+
+    public void setKeepExpanded(boolean isKeepExpanded) {
```
Comment (antares1470) [inline CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeData.java:143]: final

**Review by antares1470 — CHANGES_REQUESTED** [review]
If you're able to add a whatsnew entry as well that would be great

**Review by antares1470 — APPROVED** [review]

File: CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java [hunk 2]
```diff
@@ -675,6 +676,10 @@ private void createMultiValuePane(final AttributeData attribute, final TitledPan
             }
         });
         attributePane.setContent(dataAndMoreButtonBox);
+
+        //attributePane TitledPane EXPAND and COLLAPSE events
+        attributePane.expandedProperty().addListener((observable, wasExpanded, isNowExpanded)
```
Comment (arcturus2) [inline CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java:681]: I'm not sure if there is a way to remove the listener - I couldn't find a place to do it in this class.

**Review by arcturus2 — APPROVED** [review]
Works great.

**Pull request merged**